go.sum

```diff
@@ -124,6 +124,8 @@ github.com/buildkite/interpolate v0.0.0-20200526001904-07f35b4ae251 h1:k6UDF1uPY
 github.com/buildkite/interpolate v0.0.0-20200526001904-07f35b4ae251/go.mod h1:gbPR1gPu9dB96mucYIR7T3B7p/78hRVSOuzIWLHK2Y4=
 github.com/buildkite/roko v1.0.0 h1:JhL50LwTjjAY9CR+bK9zI0NeIEXCRDBVuNvIdYaO11c=
 github.com/buildkite/roko v1.0.0/go.mod h1:b3U4GZW/n8GbCh9+pIFLvdY+B+jlh5zOgS8s3WdELbs=
```

Comment:
I don't know how go packages work, but I would have assumed these lines would disappear?

Reply:
yeah, good call ahah! i forgot to run `go mod tidy`